index.html

```diff
@@ -65,7 +69,7 @@ <h4 class="font-h4"><br>User Experience <br>Design</h4>
 					<div class="text-exp">
 						<h4> Python Django-Intern, <span class="h4">Rights Soft Solutions 2020</span> </h4>
 					</div>
-					<div class="text-exp">
+					<!-- <div class="text-exp">
```

Comment:
Remove commented code. Also, use tab for indent and change the tab size two two in you vsCode.